src/components/Utils/filters.js

```diff
@@ -0,0 +1,25 @@
+/* eslint import/prefer-default-export: 0 */
```

Comment:
Awesome that we separate this into a util function. Would be better to have it unit tested.

Reply:
done ! added unit tests